src/libraries/math/SafeMathInt256.sol

```diff
@@ -54,4 +54,22 @@ library SafeMathInt256 {
     function getInt256Max() internal constant returns (int256) {
         return INT256_MAX;
     }
+
+    function safeToMultiply(int256 a, int256 b) internal constant returns (bool) {
+        int256 c = a * b;
+        if (a == 0 || c / a == b) {
+            return true;
+        }
+        return false;
+    }
+
+    // Float [fixed point] Operations
+    function fxpMul(int256 a, int256 b) internal constant returns (int256) {
+        require(safeToMultiply(a, b));
+        return a * b / 10 ** 18;
+    }
+
+    function fxpDiv(int256 a, int256 b) internal constant returns (int256) {
+        return a * 10 ** 18 / b;
```

Comment:
This library should be changed to match the style of `SafeMathUint256.sol` and have `fxpMul` and `fxpDiv` just be composition of `mul` and `div`.  That way we are covered from underflow/overflow issues that are not currently accounted for in these and we avoid having to extract out `safeToMultiply`.